lib/services/elasticsearch.js

```diff
@@ -339,13 +339,27 @@ class ElasticSearch extends Service {
    * Create a new document to ElasticSearch, or replace it if it already exist
    *
    * @param {Request} request
+   * @param {boolean} injectKuzzleMeta
    * @returns {Promise} resolve an object that contains _id
    */
-  createOrReplace(request) {
+  createOrReplace(request, injectKuzzleMeta = true) {
+    let kuzzleMeta = undefined;
+
     const
       esRequest = initESRequest(request, ['refresh']),
       userId = this._getUserId(request);
 
+    if (injectKuzzleMeta) {
+      kuzzleMeta = {
+        author: userId,
+        createdAt: Date.now(),
+        updatedAt: Date.now(),
+        updater: userId,
+        active: true,
+        deletedAt: null
+      };
+    };
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this empty statement. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3AS1116)